kuma/static/js/wiki.js

```diff
@@ -148,7 +154,7 @@
     /*
         Syntax highlighting scripts
     */
-    if($('article pre').length && ('querySelectorAll' in doc)) (function() {
+    if($('article pre').length && ('querySelectorAll' in doc)) {
```

Comment:
Yes, it looks like you are going to want to keep it. But you can nest it inside the outer if statement

```
if ($('article pre').length && ('querySelectorAll' in doc)) {
    (function() {
        ...
    })();
}
```